pkg/controller/workflow/result.go

```diff
@@ -7,8 +7,9 @@ import (
 )
 
 const (
-	DefaultRetry   = time.Second * 10
-	DefaultTimeout = time.Minute * 20
+	DefaultRetry                    = time.Second * 10
+	StandaloneResourceRequeuePeriod = time.Minute * 15
```

Comment:
We said we would make this configurable down to 5 mins minimum, but not advertised. We can do it on a follow up.

Reply:
Thanks for the heads up, I have fixed the TD to reiterate what was said in the scope, this should be configurable before release and 5 minutes is the minimum value allowed.